### ISSUE TITLE
Suggestion: Support Braces for building balconies etc. 

### ISSUE THREAD

**Issue by KRDucky** [EpochModTeam/Epoch#934]
It would be nice to have brace supports that can be built for placing balconies. Something like a diagonal  support brace or something.

**Comment by 82ndab-Bravo17** [comment]
You can already build balconies either by using temporary supports or placing floor pieces within each other and building out. Once the balcony is in place any unneeded pieces can be removed and the balcony will stay there without an issue.

**Comment by Ignatz-HeMan** [comment]
Building without Physics: https://github.com/EpochModTeam/Epoch/blob/experimental/Sources/epoch_config/Configs/CfgEpochClient.hpp#L113
with half / quarter Floors should do the job here...
At least you can support the Floors and later remove the foundations (as @82ndab-Bravo17 already mentioned)
